client/Dockerfile-5.12

```diff
@@ -2,6 +2,15 @@ FROM ubuntu:xenial
 
 MAINTAINER Roeland Jago Douma <roeland@famdouma.nl>
 
+# Run 'docker build' with '--build-arg BUILD_QT=1' to build Qt from source (default: not set)
```

Comment:
I'm not asking to change this in that PR, I'm just trying to solve what's still a mystery to me. :-)

Why are we not building it in the image always? Why are we building it once and downloading this separately? I'd expect that build to be a one time pain anyway, it's not the case?

Reply:
See here:

https://github.com/nextcloud/docker-ci/blob/f7700ce/client/Dockerfile-5.12#L115-L122

;-)

Docker Hub has a forced timeout for jobs, they usually terminate after around 240 minutes. So I had to come up with a solution to get us an image anyway ;)

Reply:
Ah annoying indeed... And what about making our own base image that we upload? or Docker hub forces us to let it build itself?

Reply:
Yepp ;/
Might be possible to do it this way, that just would take more time to tinker with again ;)